### DISCUSSION TITLE
Tutorial for latest version

### DISCUSSION THREAD

**Discussion by rehjaeger** [arkypita/LaserGRBL#2338] [Q&A]
I just accidentally updated to the latest version, having been running 5.1.something, and the interface of the new version is extremely different. I’ve been looking for a tutorial for the new version and haven’t found one yet - I was pretty familiar with the older version but this one appears so strange and with so many unfamiliar features I’m afraid to do any engraving until I figure it out. If there isn’t a tutorial I may have to delete it and redownload the older version I’m familiar with. Can anyone help?

**Comment by arkypita** [comment]
UI is basically the same, colors and icons are changed but everything is in the same position. If you want old color scheme, just change it from the menu "color". If you want the old icons go to menu "grbl" and "settings", last tab you will find an option called "legacy icons"

**Reply by rehjaeger** [reply]
Thank you very much - I sincerely appreciate your taking the time to respond - I was getting really uneasy about trying to do any engraving after the update.

**Comment by ReggiePye** [comment]
I wanted to add some buttons to the new version and when I tried to insert my own icon for the button I ran into a problem.
I changed to using legacy icons and then I was able to use the image that I had created for that button.
I then switched back to the new icons and the icons I had just made appeared with the new design versions.

Added this to try and help anyone else that wants to use their own images for buttons.
